.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F4A1 Feature request"
+about: "Suggest an idea for this project \U0001F914"
+title: ''
+labels: Feature request
+assignees: ''
+
+---
+
+#### Context
+
+What are you trying to do and how would you want to do it differently? Is it something you currently you cannot do? Is this related to an issue/problem?
```

Comment:
"Is it something you currently you cannot do?" -> "Is it something you currently cannot do?"

"Is this related to an issue/problem?" what data are we trying to gather for this question? Do we want the requester to provide a Github issue number?

Reply:
1. Fixed the typo :)
2. Yeah, taken from: https://github.com/nock/nock/issues/new?template=02_feature_request.md, my guess is these questions are to get as much context as possible about the feature request